examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,64 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+$ oc cluster up
+
+If you do not use oc cluster up, ensure the imagestreams are registered in the openshift namespace, as well as the
+jenkins template represented by jenkinstemplate.json:
+
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-centos7.json -n openshift
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/pipeline/jenkinstemplate.json -n openshift
+
+2) login as a normal user (any username is fine)
+
+$ oc login
+
+3) create a project for your user named "pipelineproject"
+
+$ oc new-project pipelineproject
+
+4) run this command to instantiate the template which will create a pipeline buildconfig and 
+some other resources in your project:
+
+$ oc new-app -f pipelinetemplate.json
+
+At this point if you run "oc get pods" you should see a jenkins pod, or at least a jenkins-deploy pod. (along with other items
+in your project)  This pod was created as a result of the new pipeline buildconfig being defined.
+
+Note: this template grants the edit role to the default service account for your project so jenkins can manipulate your project.  
+If you do not use this template to create your pipeline buildconfig, you need to grant edit permission to the default service
+account:
+
+$ oc policy add-role-to-user edit -z default -n pipelineproject
+
+5) View/Manage jenkins
+Once the jenkins pod is running, determine the jenkins service ip ("oc get svc") and go to it in your browser on port 80.  Do not confuse
+it with the jenkins-jnlp service.
+
+The login/password are admin/password.
+
+Go to Manage Jenkins -> Configure System
+
+Under the kubernetes plugin in the Cloud section, change the Kubernetes Namespace to "pipelineproject".
```

Comment:
i haven't tested it.  and not sure what the behavior is since the config still specifies "default" as the value right now, so presumably that overrides whatever defaulting logic he put in.  Let's leave it for now, we can always remove it later.